webviz_subsurface/_abbreviations/reservoir_simulation.py

```diff
@@ -30,7 +30,7 @@ def simulation_vector_base(vector: str) -> str:
     """Returns base name of simulation vector
     E.g. WOPR for WOPR:OP_1 and ROIP for ROIP_REG:1
     """
-    return vector.split(":", 1)[0].split("_")[0]
+    return vector.split(":", 1)[0].split("_", 1)[0][:5] if ":" in vector else vector
```

Comment:
Despite being a one-liner, the logic is a bit hard to read for newcomers. Maybe add an example in the docstring of why e.g. `[:5]` is needed?